include/LocalHostStats.h

```diff
@@ -37,10 +37,11 @@ class LocalHostStats: public HostStats {
   /* Estimate of the number of critical servers used by this host */
   Cardinality num_dns_servers, num_smtp_servers, num_ntp_servers;
 
-  /* Estimate the number of contacted hosts using HyperLogLog */
-  struct ndpi_hll hll_contacted_hosts;
-  double old_hll_value, new_hll_value, hll_delta_value;
-  DESCounter contacted_hosts;
+  /* Estimate the number of contacted hosts,asn and country using HyperLogLog*/
```

Comment:
There's class `Cardinality` (see line 38 above) that estimates the number of DNS, SMTP and NTP servers. That class should be used/extended also to do the estimations on hosts and domain names. Indeed,  `Cardinality` already uses hll in its implementation.

Reply:
It's ok using Cardinality for our hll_contacted_domain_names at line 42, but hll_contacted_hosts was not added by us. We don't know if there is actually a reason because Cardinality was not used so we will only change ours to avoid ruining something.